src/app.js

```diff
@@ -1,22 +1,12 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
-import store from './redux/store';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Routes from './route/index';
```

Comment:
@kleva-j you don't need to add the `/index` part. I mentioned this yesterday. If you want to reference an `Index.js/jsx` file in a directory, just referencing the directory name will fetch the index file.

Reply:
I actually did that and it worked but eslint was flagging it as an error. It said `Casing of ./route does not match the underlying filesystem.eslint(import/no-unresolved)`.

Reply:
I have made updates, but the tests are failing on Circle Ci because of the missing `/Index.jsx` part.